src/Symfony/Component/Intl/ResourceBundle/LanguageBundle.php

```diff
@@ -23,6 +23,8 @@
  * @author Bernhard Schussek <bschussek@gmail.com>
  *
  * @internal
+ *
+ * @deprecated since Symfony 4.2.
```

Comment:
internal

Reply:
we need to return it from `Intl::getLanguageBundle()` still, i've updated it to `@internal to be removed in 5.0.`